docs/paper/dev/misc/debugging.mdx

```diff
@@ -3,7 +3,7 @@ slug: /dev/debugging
 description: Debugging is common when writing code. This guide outlines the common ways to debug your plugin.
 ---
 
-# Debugging your plugin
+# Debugging Your Plugin
```

Comment:
Shouldn't this be `your plugin`? like it was before

Reply:
it's a top-level heading, which were all made Upper Case